demos/TicTacToe/Classes/AppDelegate.cpp

```diff
@@ -0,0 +1,31 @@
+#include "AppDelegate.h"
+
+#include "TicTacToeScene.h"
+
+USING_NS_CC;
+
+extern const float kFrameWidth = 600;
+extern const float kFrameHeight = 600;
```

Comment:
`extern` means the value is defined elsewhere. 

Think of it this way: When you declare a function, you're implicitly declaring it extern, and promising that you'll define it elsewhere. 

It's the same thing here, except the extern has to be explicit (C++ is inconsistent about this. It's confusing until you understand what's going on). You declare the variable here, and somewhere else you define what the value is.